### PR TITLE
refactor: faster levenshtein

### PR DESCRIPTION
**Pull request by ka-weihe**
What kind of change does this PR introduce?
This PR changes the levenshtein package used in `webpack-cli.js `. 

Did you add tests for your changes?
No

If relevant, did you update the documentation?
No

Summary
[leven](https://github.com/sindresorhus/leven) is one of the slowest levenshtein packages out there and the author is trying to deceive people with shitty benchmarks where he is only comparing it slower packages. Please consider using  [fastest-levenshtein]( https://github.com/ka-weihe/fastest-levenshtein) instead which is 10x faster and much higher quality. 

Does this PR introduce a breaking change?
No

Other information

### CODE REVIEW

**Comment by ka-weihe** [comment]
what

**Comment by alexander-akait** [comment]
Please rebase, we do big refactor

**Comment by alexander-akait** [comment]
And don't look at coverage, because we send different values for webpack@4 and webpack@5, so the latest values of coverage only after all builds

**Review by alexander-akait — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -5242,6 +5242,11 @@ fast-levenshtein@^2.0.6, fast-levenshtein@~2.0.6:
   resolved "https://registry.yarnpkg.com/fast-levenshtein/-/fast-levenshtein-2.0.6.tgz#3d8a5c66883a16a30ca8643e851f19baa7797917"
   integrity sha1-PYpcZog6FqMMqGQ+hR8Zuqd5eRc=
 
+fastest-levenshtein@^1.0.12:
```
Comment (anshumanv) [inline yarn.lock:5245]: there should also be removal of `leven` from lock file right?

Comment (anshumanv) [reply yarn.lock:5245]: ah it's a dep of `jest-validator` 😞

**Review by anshumanv — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**